phac-nml/irida
# Sample filter term fix

## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.
Search filter input which has underscores in it is now treated as such. The underscores in the search term are escaped.

**To test fix:**

1) Add a project
2) Add sample with name 12_1222_5933_0 and another with 12-1222-5933-0 to this project
3) In the sample "Search" try filtering by each of these. Only the sample searched for should be displayed.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.
#513

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] User documentation updated for UI or technical changes.~
